x/ccv/types/ccv.go

```diff
@@ -76,3 +76,10 @@ func (vdt SlashPacketData) GetBytes() []byte {
 	valDowntimeBytes := ModuleCdc.MustMarshalJSON(&vdt)
 	return valDowntimeBytes
 }
+
+// Unmarshals json bytes to SlashPacketData or panics if the bytes are invalid.
+func MustUnmarshalJsonBzToSlashPacketData(bz []byte) SlashPacketData {
```

Comment:
It seems that this function is only needed for testing in only one place. Could you please remove it?